apps/admin-dashboard/app/routes/_dashboard.applications.tsx

```diff
@@ -238,18 +240,34 @@ function ApplicationDropdown({ id }: ApplicationInView) {
       {open && (
         <Table.Dropdown>
           <Dropdown.List>
-            <Dropdown.Item>
-              <Link
-                to={{
-                  pathname: generatePath(Route.UPDATE_APPLICATION_EMAIL, {
-                    id,
-                  }),
-                  search,
-                }}
-              >
-                <Edit /> Update Email
-              </Link>
-            </Dropdown.Item>
+            {searchParams.status === 'pending' && (
```

Comment:
I made the assumption that we only want to have the `Update Email` dropdown tab for 'pending' applications (not 'rejected'), as this is the current behavior pre-PR.